index.html

```diff
@@ -128,6 +135,7 @@ <h4>Multi-Post Stories<br />Gain + Glory</h4>
         </ul>
         <button class="work-button" type="button">See Project</button>
       </div>
+      </div>
       <div id="about-head">
```

Comment:
- [ ] Everything else in this design is great 👌. on the desktop version you've correctly implemented the background images in this "About Me" section. however, if you look at the mobile design there is no images of the cranes on the left and right. Please implement this last change and your project should be complete.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144668997-cea87cdd-c149-437d-bba4-44ab3d8e3b8a.png)

## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/144669029-7732b8e3-dd89-4f7f-9373-14597abaa22e.png)